src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+
+import { initializeTasks, createNewTask } from './modules/tasks.js';
+import {
+  markTaskComplete,
+  markTaskIncomplete,
+  clearCompletedTasks,
+} from './modules/status.js';
+
+const input = document.querySelector('#new-task-input');
+const submitButton = document.querySelector('.add');
+
+initializeTasks();
+
+submitButton.onclick = (e) => {
+  e.preventDefault();
+  if (input.value !== '') {
+    createNewTask(input.value);
+    input.value = '';
+  }
+};
+
+function handleCheckboxChange(e) {
+  const taskId = e.target.parentElement.dataset.id;
+  if (e.target.checked) {
+    markTaskComplete(taskId);
+  } else {
+    markTaskIncomplete(taskId);
+  }
+}
+
+function handleClearButtonClick() {
+  clearCompletedTasks();
+  const taskElements = document.querySelectorAll('.task');
+  taskElements.forEach((taskElement) => {
+    if (taskElement.classList.contains('done')) {
+      taskElement.remove();
+    }
+  });
+}
+
```

Comment:
I again appreciate how all your code is easy to understand and not messy.

-  Here, you're handling your  clear all functionality based on a class ('done'). But I think  microverse said to disassociate DOM things with the program logic.

-  So for me, I would rather you use the object's completed value (that you stored in local server) to do thae manupulations.